src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1041,7 +1042,8 @@ protected int doRun() throws FlagUsageException, IOException {
     List<String> moduleSpecs = config.module;
     List<JsonFileSpec> jsonFiles = null;
 
-    if (config.useJsonStreams) {
+    if (config.jsonStreamMode == CompilerOptions.JsonStreamMode.IN ||
```

Comment:
You can import JsonStreamMode to avoid wrting the CompilerOptions prefix every time you use the enum.

    import com.google.javascript.jscomp.CompilerOptions.JsonStreamMode;